scripts/solutions.sh

```diff
@@ -437,7 +443,8 @@ add_solution() {
     local -ra pillar=(
         "{"
         "  'orchestrate': {"
-        "    'env_name': '$NAME'"
+        "    'env_name': '$NAME',"
+        "    'webhook_enabled': '$WEBHOOK_ENABLED'"
```

Comment:
I think you want to remove the quote otherwise it will always be "true" (since `bool('false') == True`)

```suggestion
        "    'webhook_enabled': $WEBHOOK_ENABLED"
```

Reply:
indeed, the quote should be removed. However, it's weird that when I tested that the process works correctly even with the quote.